packages/telemetry/rollup.config.js

```diff
@@ -73,4 +73,84 @@ const nodeBuilds = [
   }
 ];
 
-export default [...browserBuilds, ...nodeBuilds];
+const reactBuilds = [
+  {
+    input: 'src/react/index.ts',
+    output: {
+      file: pkg.exports['./react'].browser.import,
+      format: 'es',
+      sourcemap: true,
+      banner: `'use client';`
+    },
+    plugins: [
+      typescriptPlugin({
+        typescript,
+        tsconfig: 'tsconfig.react.json'
+      }),
+      json()
+    ],
+    external: id => deps.some(dep => id === dep || id.startsWith(`${dep}/`))
+  },
+  {
+    input: 'src/react/index.ts',
+    output: {
+      file: pkg.exports['./react'].browser.require,
+      format: 'cjs',
+      sourcemap: true,
+      banner: `'use client';`
+    },
+    plugins: [
+      typescriptPlugin({
+        typescript,
+        tsconfig: 'tsconfig.react.json'
+      }),
+      json()
+    ],
+    external: id => deps.some(dep => id === dep || id.startsWith(`${dep}/`))
+  }
+];
+
+const reactNodeBuilds = [
```

Comment:
These are identical to the ones above? If so we don't need to build them. Just make the package.json exports fields point to the bundles generated above.